### DISCUSSION TITLE
Plotman Error

### DISCUSSION THREAD

**Discussion by andrew429** [Chia-Network/chia-blockchain#4524] [Support]
All, I am getting the below error when I try to force-reinstall plotman.  Does anyone have any ideas here?  When I follow the path to the file, it says the owner is "root".  I am very new to all this, so any help would be appreciated. Thank you.  Andrew

ERROR: Could not install packages due to an OSError: [Errno 13] Permission denied: '/home/xxxxx/chia-blockchain/chia-blockchain-gui/chia-blockchain/chia-blockchain/venv/lib/python3.8/site-packages/__pycache__/typing_extensions.cpython-38.pyc'
Check the permissions.

**Comment by BrandtH22** [comment]
If you are still experiencing the issues it is recommended to upgrade to the latest version of chia. Please download it from here: https://www.chia.net/downloads/

Since this issue has been open for a number of years without additional comments, we will be closing this ticket but if we have closed this ticket in error do not hesitate to reach out to us again with any followup questions, comments, or if the issue persists after an update.

```
Please note that upgrading major versions of chia generally requires the config.yaml to be recreated using these steps:
1. Stop chia
2. Rename the config or move it to a different directory
3. In CLI run chia init
4. Copy any custom settings from the old config to the new (plot drive directories, pool info, trusted node info, etc) - be careful here as yaml is very picky about formatting including those leading spaces
5. Start Chia
```

The best place to reach our support team is on Discord (https://discord.gg/chia) or by reopening this ticket.
